src/foundry/common/data/data.mjs/cardData.d.ts

```diff
@@ -0,0 +1,249 @@
+import type {
+  ConfiguredData,
+  ConfiguredFlags,
+  ConfiguredSource,
+  FieldReturnType,
+  PropertiesToSource
+} from '../../../../types/helperTypes.js';
+import type DocumentData from '../../abstract/data.mjs.js';
+import * as fields from '../fields.mjs';
+import type { CardFaceData, CardFaceDataConstructorData } from './cardFaceData.js';
+
+interface CardDataSchema extends DocumentSchema {
+  _id: typeof fields.DOCUMENT_ID;
+  name: typeof fields.REQUIRED_STRING;
+  description: typeof fields.BLANK_STRING;
+  type: FieldReturnType<
+    typeof fields.REQUIRED_STRING,
+    {
+      default: () => string;
+      validate: (t: unknown) => boolean;
+      validationError: '{name} {field} "{value}" is not a valid type';
+    }
+  >;
+  data: fields.SystemDataField;
+  suit: typeof fields.BLANK_STRING;
+  value: typeof fields.NUMERIC_FIELD;
+  back: {
+    type: typeof CardFaceData;
+    required: true;
+    default: Record<string, never>;
+  };
+  faces: {
+    type: Array<typeof CardFaceData>;
+    required: true;
+    default: [];
+  };
+  face: {
+    type: typeof Number;
+    required: true;
+    default: null;
+    nullable: true;
+    validate: (f: unknown) => boolean;
+    validationError: '{name} {field} "{value}" must have a non-negative integer value or null';
+  };
+  drawn: typeof fields.BOOLEAN_FIELD;
+  origin: typeof fields.DOCUMENT_ID;
+  width: typeof fields.POSITIVE_INTEGER_FIELD;
+  height: typeof fields.POSITIVE_INTEGER_FIELD;
+  rotation: typeof fields.ANGLE_FIELD;
+  sort: typeof fields.INTEGER_SORT_FIELD;
+  flags: typeof fields.OBJECT_FIELD;
+}
+
+interface CardDataBaseProperties {
+  /**
+   * The _id which uniquely identifies this Card document
+   * @defaultValue `null`
+   */
+  _id: string | null;
+
+  /** The text name of this card */
+  name: string;
+
+  /**
+   * A text description of this card which applies to all faces
+   * @defaultValue `""`
+   */
+  description: string;
+
+  /**
+   * A category of card (for example, a suit) to which this card belongs
+   * @defaultValue `game.system.documentTypes.Card[0]`
+   */
+  type: string;
+
+  /** Game system data which is defined by the system template.json model */
+  data: object;
+
+  /** @defaultValue `""` */
+  suit: string;
+
+  /**  */
+  value: number | undefined | null;
+
+  /** @defaultValue `{}` */
```

Comment:
I think it should just be `CardFaceData`. The default value is just a `CardFaceData` that's being initialized with `{}` but it's still an instance of `CardFaceData`.